specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -446,6 +446,14 @@
                         "type": "string"
                     },
                     "description": "Optional. The Distinguished Names for cluster user groups"
+                },
+                "aaddsResourceId": {

```

Comment:
Keep in mind – there is some risk in adding properties without incrementing the API version. This largely  depends on the service behavior. In cases of update, a down-level client may PUT/PATCH the resource omitting these added properties (which may have been set by a newer client/portal) while updating other properties. In this case, what happens to the resource? Will the behavior be understood by the user? Incrementing API version enables you to have more control in these cases – such as blocking updates by older API versions.

Reply:
This should be safe, as we don't allow PUTs or PATCHs (other than tags) on this resource after creation, and will definitely not enable such modifications until the next API version.